app/api/db/migrations/versions/2022_09_20_13_16_20_dbc5b91646a2_create_user_table.py

```diff
@@ -0,0 +1,96 @@
+"""Create user table
+
+Revision ID: dbc5b91646a2
+Revises: 64cca9571cf0
+Create Date: 2022-09-20 13:16:20.206594
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "dbc5b91646a2"
+down_revision = "64cca9571cf0"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "lk_role",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("role_id", sa.Integer(), autoincrement=True, nullable=False),
+        sa.Column("role_description", sa.Text(), nullable=False),
+        sa.PrimaryKeyConstraint("role_id"),
+    )
+    op.create_table(
+        "user",
+        sa.Column(
+            "created_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated_at",
+            sa.TIMESTAMP(timezone=True),
+            server_default=sa.text("now()"),
+            nullable=False,
+        ),
+        sa.Column("user_id", postgresql.UUID(as_uuid=True), nullable=False),
```

Comment:
what are your thoughts on the naming convention of the id being `id` rather than `[table]_id`? it seems redundant to repeat the table name again.

Reply:
I think it can help when writing queries that join across several tables to know which ID you're working with / make you less likely to make a mistake for joins. Which of these would you find more readable?

```sql
SELECT
	*
FROM
	person
	JOIN place ON place.place_id = person.place_id
	JOIN thing ON thing.thing_id = place.thing_id
```
or if each table has just ID
```sql
SELECT
	*
FROM
	person
	JOIN place ON place.id = person.place_id
	JOIN thing ON thing.id = place.thing_id
```

Reply:
the second seems more readable to me, `place.id = x.place_id` since it also reads as "place id equals table place id", vs. "place dot place id equals table place id", there's extra words that don't give me extra info.